loader/0032-loader-plugins.md

```diff
@@ -0,0 +1,328 @@
+# Summary
+
+This RFC describes plugins for the Quilt mod loader, and aims to describe the
+purpose(s), functionality, and implications of such a system. At a high level,
+loader plugins will allow developers to control the mod loading process itself,
+giving them access to powerful tools for defining, loading, and removing mods.
```

Comment:
I don't see how you'd remove a mod with the API provided, it all looks additive?